### PR TITLE
Updating caption

### PR DESCRIPTION
**Pull request by test-brian-f**
## Pull Request
This PR Updates the caption because it was out of date
This PR closed / resolves / fixes #2
 
## Done
- Created a branch
- Made a commit
- 
## To do
- [ ]  merge the branch
- [ ] delete the ## My Pull Request
 
@omoferacho

### CODE REVIEW

File: _slides/17-test-brian-f.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+In this moment, I am euphoric
```
Comment (lchafe1) [inline _slides/17-test-brian-f.md:6]: More cowbell

File: _slides/17-test-brian-f.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (lchafe1) [inline _slides/17-test-brian-f.md:7]:
```suggestion
Love cowbell
```

File: _slides/17-test-brian-f.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (craighubleygit) [inline _slides/17-test-brian-f.md:5]: This is my Caption

File: _slides/17-test-brian-f.md (hunk 1 above)
Comment (justin-crane) [inline _slides/17-test-brian-f.md:6]: Here's a good ol comment

Comment (justin-crane) [inline _slides/17-test-brian-f.md:6]:
```suggestion
In every moment, I am euphoric
```

**Review by craighubleygit — APPROVED** [review]

**Review by lchafe1 — APPROVED** [review]
More and more cowbell

**Review by pchaps — APPROVED** [review]
Completed

**Review by justin-crane — APPROVED** [review]
👍

**Pull request merged**